http/vibe/http/server.d

```diff
@@ -837,7 +837,17 @@ enum SessionOption {
 
 		See_Also: secure, Cookie.secure
 	*/
-	noSecure = 1<<2
+	noSecure = 1<<2,
+
+	/**
+    Prevents the browser from sending this cookie along with cross-site requests.
+
+    By default, the protection is `lax`. This flag allows to set it to `strict`.
+    The strict value will prevent the cookie from being sent by the browser
+    to the target site in all cross-site browsing context,
+    even when following a regular link.
+	*/
+	sameSiteStrict = 1<<3,
```

Comment:
I'd even argue to make this the default and provide an optional flag to switch back to the default behavior. Should of course be noted prominently in the change log, but it would be bad to leave the safe-by-default track here.